### ISSUE TITLE
Can I pool parameter estimates of multiple imputed data obtained through10-fold cross-validation？

### ISSUE THREAD

**Issue by shenlan17** [amices/mice#394]
I performed 10-fold cross-validation on mice，using `'ignore'` parameter in `mice` package, 5 datasets were obtained each time, and a total of 50 imputed datasets were obtained. I want to do linear regression using these 50 datasets  . 

Can I use Rubin's Rules to directly combine the effect estimates(β) of 50 datasets and then derive confidence intervals and p-values?  or only every 5 datasets ( imputed by the same training set and test set ) can be combined?

Thank you,
Liu.

**Comment by gerkovink** [comment]
@paulinavonstackelberg Could you shine your light on this, perhaps with a reference to code?
